instrumentation/kotlinx-coroutines/kotlinx-coroutines-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/kotlinxcoroutines/instrumentationannotations/WithSpanInstrumentation.java

```diff
@@ -324,13 +324,6 @@ public void visitEnd() {
               // initialize our local variables, start span and open scope
               {
                 MethodNode temp = new MethodNode();
-                // insert
-                // request =
-                // AnnotationInstrumentationHelper.createMethodRequest(InstrumentedClass.class,
-                //   instrumentedMethodName, withSpanValue, withSpanKind)
-                // context = AnnotationInstrumentationHelper.enterCoroutine(label, continuation,
-                // request)
-                // scope = AnnotationInstrumentationHelper.openScope(context)
```

Comment:
I think this shouldn't be removed

Reply:
I will revert it, do you have any particular reason to keep it around ? Ideally if we could add a comment for why it might be relevant in the future it would be great.

Reply:
My bad, I hadn't read it closely and removed because it looked like commented code, when in fact it's documenting the code we are injecting.